consul/src/test/java/com/linecorp/armeria/internal/consul/ConsulTestBase.java

```diff
@@ -58,101 +55,63 @@
  * Selectively disable Consul tests to suppress the stressful flakiness.
  */
 @FlakyTest
+@Testcontainers
+@EnabledIfDockerIsAvailable
```

Comment:
I think `@Testcontainers` already has an option for this

https://github.com/open-telemetry/opentelemetry-java/blob/ad6cbfc4b49b1c996b588a3942d665034468da21/integration-tests/otlp/src/main/java/io/opentelemetry/integrationtest/OtlpExporterIntegrationTest.java#L106

Reply:
Ahhh, thanks for the heads up, @anuraaga :bow:

Reply:
I was able to see the test results in CI with `@Testcontainers(disabledWithoutDocker = true)` as @anuraaga said.
```
./tests/test/classes
./tests/test/classes/com.linecorp.armeria.client.consul.ConsulEndpointGroupBuilderTest.html
./tests/test/classes/com.linecorp.armeria.client.consul.ConsulEndpointGroupTest.html
./tests/test/classes/com.linecorp.armeria.internal.consul.ConsulClientBuilderTest.html
./tests/test/classes/com.linecorp.armeria.server.consul.ConsulUpdatingListenerTest.html
```